src/bcf/mod.rs

```diff
@@ -303,6 +303,158 @@ impl Drop for IndexedReader {
     }
 }
 
+/// This module contains bitmask constants for `SyncedReader`.
```

Comment:
Maybe put the entire symmetric reader code in a submodule of bcf and call `sr_paring` `pairing` instead.

Reply:
Hm, although this is not Python, I like the "flat is better than nested" theme ;) If you think it's better style, I can do this.

Reply:
Yes. The indicator here is that you have to use sr_pairings for the module with the constants. Further, it is quite some complexity added to bcf, that is not necessarily part of it. Hence I think it makes sense to add it as a submodule.

Reply:
👍